BILLthebuilder/automart
# #166981921 Improve test coverage

#### What does this PR do?
* It improves test coverage
#### Description of Task to be completed?
* Adding more test cases and refactoring code
#### How should this be manually tested?
* Clone the repository
* Checkout to the `ch-test-coverage-166981921` branch
* `npm install`
* `npm test`
* All tests should pass with increased coverage from 81% - 88%
